python-sdk/nuscenes/map_expansion/fix_map.py

```diff
@@ -0,0 +1,50 @@
+import argparse
+import json
+import os
+
+from nuscenes.map_expansion.map_api import NuScenesMap
+from nuscenes.map_expansion.utils import drop_disconnected_lanes
+
+
+def fix_map(dataroot: str, map_name: str, out_suffix: str) -> None:
```

Comment:
Suggestion to rename this file and method to `drop_disconnected_lanes`.

Reply:
Resolved by deleting the file.